tnqvm/TNQVM.cpp

```diff
@@ -92,10 +93,19 @@ void TNQVM::execute(std::shared_ptr<xacc::AcceleratorBuffer> buffer,
                     const std::shared_ptr<xacc::CompositeInstruction> kernel) {
   // Get the visitor backend
   visitor = xacc::getService<TNQVMVisitor>(getVisitorName());
+  visitor->setOptions(options);
 
   // Initialize the visitor
   visitor->initialize(buffer, nbShots);
 
+  // If this is an Exatn-MPS visitor, transform the kernel to nearest-neighbor
+  if (visitor->name() ==  "exatn-mps")
+  {
+    auto opt = xacc::getService<xacc::IRTransformation>("lnn-transform");
+    opt->apply(kernel, nullptr,  { std::make_pair("max-distance", 1)});
```

Comment:
hard-coded max-distance. is this something we want to add to TNQVM initialize options?

Reply:
Currently, we don't have support for >1 distance in the visitor. 
When the gate aggregation simulation method is implemented, we'll make this a user option. 
I've added a comment to this code as well.